pages/badges/badge-single.jsx

```diff
@@ -152,11 +152,15 @@ var BadgePage = React.createClass({
       criteria = criteria.split(splitOnItems).map(s => s.trim()).filter(s => s);
     }
 
+    var desc = document.createElement("p");
+
+    desc.innerHTML = bdata.description;
```

Comment:
we do this to turn characters that Credly turned into html entities back into plain text. As these are our own badges, the risk of user generated content XSS is extremely low.